LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 commitizen
```

Comment:
This should probably read something like `2015-2018 Commitizen Contributors`

Reply:
@LinusU Thanks for the feedback, updated it just now.